### ISSUE TITLE
Equivalence in "image alternatives" 2.1 Image and media alternatives Developing Guideline 2.1.1 Image alternatives

### ISSUE THREAD

**Issue by brett-oppegaard** [w3c/wcag3#370]
Equivalence (or "Equivalent") information on image alternatives has been a greatly abused standard under WCAG 2.0. This problem has its legacy in "alt text" originally being a computer science adaptation (not a disability studies-oriented suggestion) to early-days loading problems with images, providing a text label to alert users that, yes, a photo was supposed to be in that outlined box. 

But in today's world, what happens in practice most often is that "alt text" becomes a legal compliance standard for "equivalent," and alt text often reads like a cryptic file name. For example, an alt text / equivalent "standard" of "dog32234.jpg" means an image on a website is legally complaint in all ways, but, of course, it fails miserably any common-sense idea about equivalent information transfer. We don't know if this is a Great Dane or Chihuahua; or it could be Dog, The Bounty Hunter; or it could be a slang label for a person who is a friend or a dastardly creep; what is the "dog" doing in the park? What does the park look like? None of those questions have certain or even probable answers, and they all would include a great deal of nuance, if described properly, as well as equivalent informational content. When I shared the alt text of "dog32234.jpg," did you picture a French Bulldog, named Pinky, on a sandy beach in Hawaii, with a beachball in her mouth, running alongside the waves? Or did you picture a Rottweiler, named Buddy, sprawled out on a grassy field, with the sun on his smiling face, while he's beginning to roll on his back? Or something else entirely? That's what I mean. The image is distinct in some way. It was chosen for some rhetorical purpose by the page designer. It is filled with information; what kind of beachball? A blue and white beachball, or a rainbow colored beachball, or a black beachball; those all change the information and the subtextual and unspoken communication happening between viewer of the image and the creator of the image / designer of the page.

Why does all of that matter? If it doesn't matter, then I suggest we turn off all images on the Internet for a day, and provide people only with the alt text that exists right now, and determine if that has any material effect on the use and understanding of the web. If you think it would, then I think you would agree with me that equivalent needs to be much, much better defined, and also extended in guidance to standards for describing people, maps, tables, charts, data visualizations, etc., beyond just the title of the image, and a one-sentence overview. Think about how much of the current Internet is inaccessible, in terms of just new photographs, maps, and data visualizations, and how that inaccessibility is EXPONENTIALLY increasing every single day because of a lack of standards and legal requirements about media accessibility. Just think of Instagram, as an example, where more than 1 BILLION mostly inaccessible photos are being uploaded a day. The current "requirement" for using Instagram makes adding even cryptic alt text a hidden process; Instagram is pushing people to use GenAI; the GenAI is providing very limited descriptions, including of people, like Might be a man, wearing glasses. That is not equivalent to what a sighted person can see, of course. ... So Instagram, Facebook, etc., are perpetuating and growing media inaccessibility based on their designs and lack of accessibility support. That's purposeful, because they don't have to, and NOBODY, including WCAG guidelines, are even suggesting that they need to do this in any meaningful way. 

Please change that trajectory by defining "equivalent" in a meaningful way, in relation to information exchange (academic literature does it, so you can borrow from those scholars). 

 

Commenting on:

2.1 Image and media alternativesDeveloping
Guideline 2.1.1 Image alternatives

Foundational requirement: Equivalent text alternativeDeveloping
Equivalent

text alternatives
are available for [images](https://www.w3.org/TR/wcag-3.0/#dfn-image) that convey [content](https://www.w3.org/TR/wcag-3.0/#dfn-content).

**Comment by brett-oppegaard** [comment]
I appreciate your point, but I don't think that happens generally in
practice. ... If the spirit is there (that's arguable, but I'm not trying
to argue about it), there is a radically extreme compliance/enforcement
issue with all of these "WCAG compliant" websites; check a few websites,
and why is "dog32234.jpg" in use almost everywhere? Use your screenreader
or right-click on images on any random website outside the largest
corporations with armies of alt text writers (who, yes, will put in "Dog in
park." Then, "Inspect" the HTML, and in my experiences, I've found with
very, very high probability that I will find file names or one-line "Dog in
Park" alt text on virtually every site on virtually every page. Per Perkins,*
less than 1 percent *of the web is compliant,
https://www.perkins.org/resource/how-write-alt-text-and-image-descriptions-visually-impaired/,
per AccessEyes (https://www.audioeye.com/post/accessibility-statistics/), ~*3
percent *of the web is accessible; 4 percent according to WebAim (
https://webaim.org/projects/million/) and those are the "top" websites,
even judged by those low standards, and so on. ... So I think we really
need to focus in WCAG 3 on the definition of "equivalent" and "compliant."
I also would refer back to "understandable" and "robust" in WCAG 2
guidelines; do you think those are being generally met by most websites? If
not, is the problem with the wording, the spirit, the compliance, the
enforcement, the loopholes, or what? Or, if you don't think there are
problems with it, I'm interested to hear that debate as well. Thanks for
your response!

**Comment by yatil** [comment]
WCAG can only define what conformance is, it cannot enforce conformance. That's what laws and policies are for. And if your alternative text does not meet the success criterion, and "dog123.jpg" generally doesn't, then you don't conform to WCAG.

**Comment by brett-oppegaard** [comment]
So how does WCAG respond to claims in its name of conformance, like "This
website is WCAG compliant," when it's not, per the examples provided? Can
it create a cease and desist letter for using that label, which everyday
consumers can send when they find violations (if so, is 1 violation enough,
2, 5, 10, 100)? Just wondering where the buck stops on that one? Because I
highly doubt there will ever be a government-funded WCAG police force
roaming the internet for it, shutting down non-compliant websites. If WCAG
is legally "required" but not legally enforced, what real value does it
have in practice? ... Also, I think that if WCAG fails to properly frame
the bar for compliance, with lots of room for interpretation, loopholes,
and fuzzy definitions, then most corporations (and people) will continue to
reach for the lowest and most generous benchmarks of compliance possible. I
was at one of the world's biggest tech companies last week, and was told
about meetings there, where the overriding question regularly is "how does
this company initiative address legal compliance?" Not, how does this
company initiative position us as thought leaders in accessibility, or
innovators, but more like, where is the lowest bar, let's hit that, and
only respond when the bar is raised and not meeting it will be a
public embarassment. ... But, of course, too rigid of guidelines would be
counterproductive, too, because so few people would meet them (i.e. 95+
percent of websites already are failing for people who are blind). So is
this really an exercise in "moving the ball forward," or in significantly
raising standards to the culturally appropriate level for our time, or in
pollyannaish hopefulness, or something else?

**Comment by yatil** [comment]
This is legally enforced in different ways, for example as a customer you have rights in case a website is not accessible. This is mostly civil rights, so you can sue a website for not being accessible and WCAG is then used as the standard to measure against.

In the EU the European Accessibility Act does require companies legally to meet WCAG 2.1 as part of the EN 301 549. The countries are required to create agencies that ensure that everyone in the market that falls under the EAA meets its requirements. There are fines for non-compliance. (This has started only this year, there will be a lot happening in the next 12 to 18 months. Also this is simplified, of course.)

W3C does not enforce standards like WCAG, it's the place where stakeholders come together and agree what the standards should be. Keeping to the standard is technically optional, but companies have an interest in operability. W3C has neither the budget nor the intent to review billions of websites.

Hope that helps.
